content/millennium/r4/encounters/encounter.md

```diff
@@ -58,12 +58,12 @@ The following fields are returned if valued:
 
 All URLs for custom extensions are defined as `https://fhir-ehr.cerner.com/r4/StructureDefinition/{id}`
 
- ID                                                        | Value\[x] Type      | Description
------------------------------------------------------------|---------------------|----------------------------------------------------------------
- `client-organization`                                     | [`Reference`]       | The financially responsible organization.
- `estimated-financial-responsibility-amount`               | [`Money`]           | The estimated amount to be collected for the encounter.
- `estimated-financial-responsibility-not-collected-reason` | [`CodeableConcept`] | The reason no estimated amount is collected for the encounter.
- `payment-collection-status`                               | [`CodeableConcept`] | The status of the payment collection for the encounter.
+ ID                                                        | Value\[x] Type                                                   | Description
+-----------------------------------------------------------|------------------------------------------------------------------|----------------------------------------------------------------
+ `client-organization`                                     | [`Reference`](https://hl7.org/fhir/r4/references.html#Reference) | The financially responsible organization.
```

Comment:
Why are we adding the link for [`Reference`] here instead of at the bottom of the file? Is it because it could get confusing with [`reference`]?

Reply:
reference link (i.e. Markdown reference-style link) text is not case sensitive so it's not possible to have two different reference links for words that differ only in case: https://daringfireball.net/projects/markdown/basics